base/server/python/pki/server/deployment/__init__.py

```diff
@@ -2579,6 +2570,23 @@ def import_system_certs(self, nssdb, subsystem):
 
         self.import_cert_chain(nssdb, subsystem)
 
+        # Import admin cert after importing the cert chain so that
+        # it can be verified.
+
+        if subsystem.name in ['kra', 'ocsp']:
```

Comment:
I'm actually not sure why in the original code the admin cert import was only done for KRA & OCSP and not for TKS & TPS. It's possible that this code is actually redundant since the same tests for TKS & TPS seem to be working just fine without it. We can clean it up separately later.